src/applications/vaos/new-appointment/redux/selectors.js

```diff
@@ -259,7 +259,7 @@ export function selectProviderSelectionInfo(state) {
     address: selectVAPResidentialAddress(state),
     typeOfCareName: typeOfCare.name,
     communityCareProviderList:
-      communityCareProviders[`${sortMethod}_${typeOfCare.ccId}`] || [],
+      communityCareProviders[`${sortMethod}_${typeOfCare.ccId}`],
```

Comment:
Removed the default here so that we can separate the undefined state (providers are still being loaded or there's an error) from the 0 providers found state (an empty list)